### PR TITLE
Do not print credentials into build log

### PR DESCRIPTION
**Pull request by legionus**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1694861
Tests will be in a follow up PR for the origin repo,

/assign @adambkaplan

### CODE REVIEW

File: pkg/build/builder/source.go [hunk 1]
```diff
@@ -387,7 +387,7 @@ func extractSourceFromImage(ctx context.Context, dockerClient DockerClient, stor
 
 	if auths != nil {
 		for registry, ac := range auths.Configs {
-			glog.Infof("Setting authentication for registry %q using %#v", registry, ac)
+			glog.Infof("Setting authentication for registry %q using %q.", registry, ac.ServerAddress)
```
Comment (adambkaplan) [inline pkg/build/builder/source.go]: Log level should be V(5) for this (debug). - `glog.V(5).Infof(...)`
There are similar log statements to this which should also be V(5):

https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L55-L60
https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L102-L109

Comment (legionus) [reply pkg/build/builder/source.go]: @adambkaplan fixed

**Comment by adambkaplan** [comment]
/retest

Hitting the Unauthorized bug

**Pull request merged**